src/main/java/com/level11data/databricks/session/WorkspaceSession.java

```diff
@@ -83,82 +84,50 @@ public WorkspaceSession() throws DatabricksClientConfigException {
     }
 
     private void createEncryptedHttpSession() {
-        //create secure https client session
         ClientConfig clientConfig = new ClientConfig();
-        clientConfig.register(new JacksonFeature());
-        clientConfig.connectorProvider(new HttpUrlConnectorProvider());
-        SSLContext sslContext = SslConfigurator.newInstance().securityProtocol(SECURITY_PROTOCOL).createSSLContext();
-        System.setProperty("https.protocols", SECURITY_PROTOCOL);
-        System.setProperty("jdk.tls.client.protocols", SECURITY_PROTOCOL);
-
-        _httpClient = ClientBuilder.newBuilder()
-                .sslContext(sslContext)
-                .withConfig(clientConfig)
-                .build();
+        clientConfig.register((Object)new JacksonFeature());
+        clientConfig.connectorProvider((ConnectorProvider)new HttpUrlConnectorProvider());
+        SSLContext sslContext = SslConfigurator.newInstance().securityProtocol("TLSv1.2").createSSLContext();
+        System.setProperty("https.protocols", "TLSv1.2");
+        System.setProperty("jdk.tls.client.protocols", "TLSv1.2");
+        _httpClient = ClientBuilder.newBuilder().sslContext(sslContext).withConfig((Configuration)clientConfig).build();
```

Comment:
looks like here only change here is replacing constants with string tokens, verify line by line and revert

Reply:
in general, most changes in this file look like decompile related